### DISCUSSION TITLE
Export data from Qfield Offline

### DISCUSSION THREAD

**Discussion by Giovanninho00** [opengisch/QField#2974] [Q&A]
Good morning,

I have a problem with Qfield cloud, I downloaded the project from the cloud and I worked on it on the phone, after a few days I have deleted that project from the cloud, now is it possible to extract the data of the project saved on the phone or have I have lost all my work?

**Comment by nirvn** [comment]
@Giovanninho00 , in upcoming QField 2.2 (beta circa end of June), you will be able to open your locally mirrored cloud project folder and extract data. Hold on a bit longer :)

If you need it sooner, you can use android tools to backup QField app, the internal cloud projects folder should have been captured in the created backup file.
